roles/openshift_health_checker/openshift_checks/docker_storage.py

```diff
@@ -143,7 +143,7 @@ def _get_vg_free(self, pool, task_vars):
                 "so the available storage in the VG cannot be determined.".format(pool)
             )
         vg_name = match.groups()[0].replace("--", "-")
-        vgs_cmd = "/sbin/vgs --noheadings -o vg_free --select vg_name=" + vg_name
+        vgs_cmd = "/sbin/vgs --noheadings -o vg_free --units g --select vg_name=" + vg_name
```

Comment:
We use the parsing anyway to read the docker driver info, where we have no options to tell it what to give us.

Percentage of free space is certainly confusing...